src/zrc/api/serializers.py

```diff
@@ -45,8 +45,26 @@
 
 
 # serializers for betrokkene identificatie data used in Rol api
-class RolNatuurlijkPersoonSerializer(serializers.ModelSerializer):
+class RenameModelSerializer(serializers.ModelSerializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # iterating through all fields in a nasty way to keep their order
+        fields_count = len(self.fields)
```

Comment:
is `self.fields` not an `OrderedDict`?

Reply:
it's a `BindingDict` based on `OrderedDict`

Reply:
hrm, does this retain the order? I'd expect new keys to be added at the end, which is what happens when you do the rename. Or maybe it treats the non-rename as new key insertions and removes the key only to re-insert it at the end?

Reply:
during this loop I remove all the items from `self.fields` ([line](https://github.com/VNG-Realisatie/gemma-zaakregistratiecomponent/blob/d802250ff25aa52705d91b99392eee57e6005d77/src/zrc/api/serializers.py#L55))
and after that I add them again - with or without renaming
I couldn't make up a better way to change `self.fields` preserving the order

Reply:
yeah it doesn't look very attractive but it does the job and I can't think of a more elegant way myself